ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -75,3 +75,41 @@
 .contenttreeWindow {
     z-index: 70001 !important;
 }
+
+// Listingblock
+.sl-block.ftw-simplelayout-filelistingblock {
+  table {
+    tr {
+      th {
+        white-space: nowrap;
+        text-align: left;
+        &.header-getObjSize {
+          text-align: right;
+        }
+      }
+      td {
+
+        &.column-modified {
+          width: 85px;
+        }
+
+        &.column-portal_type {
+          width: 25px;
+          padding-right: 0;
+        }
+
```

Comment:
@bierik 
Nice catch, it seems like the "column-getContentType" was removed and replaced with "column-portal_type". I remove this.

column-modified => Bearbeitet
column-getObjSize => Grösse
column-portal_type => Typ

All other columns do not matters, since they don't have any specific styles. 

FYI:
`portal_type` usually means `File`, `Link`, `ContentPage`, etc. `content_type` means `PDF`, `XLSX`, etc.